programs/openbook-v2/src/accounts_ix/create_market.rs

```diff
@@ -3,18 +3,21 @@ use anchor_lang::prelude::*;
 use anchor_spl::token::{Mint, TokenAccount};
 
 #[derive(Accounts)]
-#[instruction(market_index: MarketIndex)]
+#[instruction()]
 pub struct CreateMarket<'info> {
     #[account(
         init,
-        seeds = [b"Market".as_ref(), payer.key().to_bytes().as_ref(), &market_index.to_le_bytes()],
-        bump,
         payer = payer,
         space = 8 + std::mem::size_of::<Market>(),
     )]
     pub market: AccountLoader<'info, Market>,
+    #[account(
+        seeds = [b"Market".as_ref(), market.key().to_bytes().as_ref()],
+        bump,
```

Comment:
you should use the stored bump here in order to reduce CU.
alternatively, given that it seems that you're also saving the pubkey inside the market (please see following comments), you can directly check the address with `has_one` constraints